test/integration/imagechange_buildtrigger_test.go

```diff
@@ -64,6 +69,9 @@ func TestSimpleImageChangeBuildTrigger(t *testing.T) {
 		t.Fatalf("Expected build with base image %s, got %s", "registry:8080/openshift/test-image:ref-2", newBuild.Parameters.Strategy.DockerStrategy.BaseImage)
 	}
 
+	event = <-watch2.ResultChan()
+	event = <-watch2.ResultChan()
```

Comment:
@smarterclayton where is it updating it twice?  adding @soltysh since it updated it zero times until we merged in his code.